ReactAndroid/src/main/java/com/facebook/react/views/text/ReactFontManager.java

```diff
@@ -71,6 +83,15 @@ public static ReactFontManager getInstance() {
     return typeface;
   }
 
+  public void loadFont(Context context, int fontId) {
```

Comment:
should rename to `addCustomFont()` to make it clearer

Reply:
Can you also add comment for how to call this method? E.g. what you did for RNTester. Basically document how one can install a custom font support in their apps.